lib/psql.js

```diff
@@ -51,6 +52,7 @@ function * exec (db, query) {
   let configs = bastion.getConfigs(db)
 
   yield bastion.sshTunnel(db, configs.dbTunnelConfig)
+  debug('Running query: %s', query.trim())
```

Comment:
Curious why not push down into `execPsql`? It's only called from here currently, so it's moot, but I guess since that's a lower-level path we may want to use it for things that we don't want to log at some point?